beacon_node/beacon_chain/tests/block_verification.rs

```diff
@@ -991,6 +992,7 @@ async fn block_gossip_verification() {
     let (chain_segment, chain_segment_blobs) = get_chain_segment().await;
 
     let block_index = CHAIN_SEGMENT_LENGTH - 2;
+    let cgc = 0;
```

Comment:
I think CGC isn't relevent for gossip verifcation, but we could still use  `spec.custody_requirement` here - the tests are run for every fork so it would be good if the functions are called in a similar way to production.

Reply:
2ba26543e5f039b1ee9774bb1c4c2f21dfa2a5e1